source/geod24/LocalRest.d

```diff
@@ -931,6 +931,45 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
         }
 }
 
+/// test for scoped delegates
+unittest
```

Comment:
Can you move this towards the end please ? The first example is meant to be / stay a simple usage example. Also I wouldn't make this a documented test